foerder.html

```diff
@@ -226,12 +226,18 @@
 				formDict[pair[0]] = pair[1];
 			}
 
-			var jsonMsg = JSON.stringify(formDict);
```

Comment:
Agreed. Maybe msgpack would be a viable option. I'll look into getting msgpack into and out of qrcodes.
I think we should still eyeball each entry. At the very least we have to make sure the qrcode data matches up with the printed out data.